create_shards.c

```diff
@@ -534,3 +603,30 @@ IntegerToText(int32 value)
 
 	return valueText;
 }
+
+
+/*
+ *	GetSupportRoutine helps to find the support routine given a column, an access method
+ *	and id of a support function. This function returns InvalidOid if there is no support
+ *	routine associated with the data type of the column or there is no default operator
+ *	class for the data type of the column.
+ */
+Oid
+GetSupportRoutine(Oid columnOid, Oid accessMethodId, int16 supportFunctionNumber)
```

Comment:
At first I was going to say `columnOid` is misleading because this function doesn't have to know that the `typeId` it receives belongs to a column, but seeing that here gave me an idea…